src/components/forms/fragments/RegistrationAdditionalInfo.js

```diff
@@ -8,29 +8,29 @@ import type { FormFragmentProps } from '../../../types/form';
 
 const RegistrationAdditionalInfo = ({ prefix }: FormFragmentProps) => (
   <Fragment>
-    <FormattedMessage tagName="h3" id="page.boat.form.section.boat_additional_info" />
+    <FormattedMessage tagName="h3" id="form.registered.header.additional_info" />
```

Comment:
Should this be `registration_additional_info` or some similar? I feel both works, as it's only used in boat info, but might be miss leading 🤔